server/routes/authRoutes.js

```diff
@@ -5,21 +5,21 @@ const User = mongoose.model("User");
 
 const jwt = require("jsonwebtoken");
 
- router.post("/signup", async (req, res) => {
-	const { email, password } = req.body;
+router.post("/signup", async (req, res) => {
+	const { email, password, name } = req.body;
 	try {
-		const user = new User({ email, password });
+		const user = new User({ email, password, name });
 		await user.save();
-		const token = jwt.sign({ userId: user._id }, "MY_SECRET_KEY");
+		const token = jwt.sign({ userId: user._id, name }, "MY_SECRET_KEY");
 		res.send({ token });

```

Comment:
save the field name when signing up and store into the token so we can access the user name in the token